src/model/sticker.rs

```diff
@@ -0,0 +1,88 @@
+use crate::model::{
+    id::{GuildId, StickerId, StickerPackId},
+    user::User,
+};
+
+/// A sticker sent with a message.
+///
+/// Bots cannot send stickers.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[non_exhaustive]
+pub struct Sticker {
+    /// The unique ID given to this sticker.
+    pub id: StickerId,
+    /// The unique ID of the pack the sticker is from.
+    pub pack_id: Option<StickerPackId>,
+    /// The name of the sticker.
+    pub name: String,
+    /// Description of the sticker
+    pub description: String,
+    /// For guild stickers, a unicode emoji representing the sticker's
+    /// expression. For nitro stickers, a comma-separated list of related
+    /// expressions.
+    pub tags: String,
```

Comment:
You should make it an array to keep consistency and to make it easier to handle -which would mean that the array would be of length 1 if it is a guild owned one-